users/brett/brett.c

```diff
@@ -0,0 +1,118 @@
+#include "brett.h"
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch(keycode) {
+    case FAT_ARROW:
+      if (record->event.pressed) {
+        SEND_STRING("=>");
+      }
+      return false;
+      break;
+    case SKINNY_ARROW:
+      if (record->event.pressed) {
+        SEND_STRING("->");
+      }
+      return false;
+      break;
+    case REVERSE_ARROW:
+      if (record->event.pressed) {
+        SEND_STRING("<-");
+      }
+      return false;
+      break;
+    case CONCAT:
+      if (record->event.pressed) {
+        SEND_STRING("<>");
+      }
+      return false;
+      break;
+    case MAP:
+      if (record->event.pressed) {
+        SEND_STRING("<$>");
+      }
+      return false;
+      break;
+    case MAP_FLIPPED:
+      if (record->event.pressed) {
+        SEND_STRING("<#>");
+      }
+      return false;
+      break;
+    case FLAP:
+      if (record->event.pressed) {
+        SEND_STRING("<@>");
+      }
+      return false;
+      break;
+    case PIPE:
+      if (record->event.pressed) {
+        SEND_STRING("|>");
+      }
+      return false;
+      break;
+    case ALT:
+      if (record->event.pressed) {
+        SEND_STRING("<|>");
+      }
+      return false;
+      break;
+    case APPLY:
+      if (record->event.pressed) {
+        SEND_STRING("<*>");
+      }
+      return false;
+      break;
+    case AND:
+      if (record->event.pressed) {
+        SEND_STRING("&&");
+      }
+      return false;
+      break;
+    case OR:
+      if (record->event.pressed) {
+        SEND_STRING("||");
+      }
+      return false;
+      break;
+    case BIND:
+      if (record->event.pressed) {
+        SEND_STRING(">>=");
+      }
+      return false;
+      break;
+    case BIND_FLIPPED:
+      if (record->event.pressed) {
+        SEND_STRING("=<<");
+      }
+      return false;
+      break;
+    case DOUBLE_COLON:
+      if (record->event.pressed) {
+        SEND_STRING("::");
+      }
+      return false;
+      break;
+    case VOID_LEFT:
+      if (record->event.pressed) {
+        SEND_STRING("<$");
+      }
+      return false;
+      break;
+    case VOID_RIGHT:
+      if (record->event.pressed) {
+        SEND_STRING("$>");
+      }
+      return false;
+      break;
+    case FLASH:
+        if (record->event.pressed) {
+            SEND_STRING("sudo -E make hhkb:brett:dfu");
```

Comment:
Using `sudo` is bad.   Adding udev rules is better:
https://docs.qmk.fm/#/faq_build?id=linux-udev-rules

Also, if you include `version.h`, you can just do this: 
```suggestion
            SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP ":flash" SS_TAP(X_ENTER));
```

That will run make for the current keyboard and keymap, and use the `:flash` target, which "auto detects" which target to actually use, based on the bootloader or architecture. 

Also, adding the `SS_TAP` at the end should also remove the need for the (un)register_code below, too.